### ISSUE TITLE
Imagick on PHP 5.6 can't read PDF files

### ISSUE THREAD

**Issue by thomasA4N** [cytopia/devilbox#714]
On PHP 5.6 Imagick can't read pdf files. It's not working with relative not even absolut path.

Example:

```
try {
	$im = new imagick();
	$im->setResolution(288,288);
	$im->readimage(realpath('test.pdf')); // relative path also doesn't works
} catch (Exception $e){
	echo $e->getMessage()."<br />\n"; // gets error "Failed to read the file"
}
```
[test.pdf](https://github.com/cytopia/devilbox/files/5034233/test.pdf)

**Comment by cytopia** [comment]
@thomasA4N can you submit some example code, I will then add it as a regression test here: https://github.com/devilbox/docker-php-fpm

**Comment by thomasA4N** [comment]
@cytopia i have updated my comment ...

**Comment by cytopia** [comment]
Figured it already, it seems to be the lack of `ghostscript`. Once your stack is running, can you do the following in the PHP container:

```
sudo apt update
sudo apt install ghostscript -y
```

This should fix it for now. Will bundle it in the containers in the next release.

**Comment by thomasA4N** [comment]
Thank you very much. I have tested that, and it's already works now. 

Now, I will have to repeat that, after every restart of devilbox, didn't i?

**Comment by cytopia** [comment]
> Now, I will have to repeat that, after every restart of devilbox, didn't i?

No, when you stop the container, the state will be preserved across restarts until you remove it explicitly (via `docker-compose rm` or `docker rm`)

**Comment by cytopia** [comment]
@thomasA4N will be fixed here: https://github.com/cytopia/devilbox/pull/711

**Comment by cytopia** [comment]
Merged and available in `v1.7.1`
